### ISSUE TITLE
Integrate with new CAC API for side panel

### ISSUE THREAD

**Issue by IceRaptor** [BattletechModders/LowVisibility#41]
KMission provided me a new API that allows control of the side panel - integrate with it.

**Comment by IceRaptor** [comment]
there two methods in API 
    CustAmmoCategories.CombatHUDInfoSidePanelHelper.SetSelfInfo(AbstractActor actor, Text text) 
    CustAmmoCategories.CombatHUDInfoSidePanelHelper.SetTargetInfo(AbstractActor actor,ICombatant target, Text text)

**Comment by IceRaptor** [comment]
Done in 8.0.0, but updates only on a refresh of the targeting HUD. Per KMission 'it would be nice' if the update reflected the current preview move state, if possible.

```
no i mean in battle :slight_smile:
eg . it is good idea to update it on MoveStatusPreview.Prefix taking Vector3 worldPos as base point instead of unit.CurrentPosition
i think
```

**Comment by IceRaptor** [comment]
Fixed in 1.0.0
